### PR TITLE
Addressed PR #27 comments

### PR DESCRIPTION
**Pull request by zhanggbj**
Hi dr.max @maximilien ,

Addressed all comments and re-submitted docs review here as the original one got a little messy. Thanks for your review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+-------------
+You can use bosh-init from BOSH community to bootstrap a pool server on SoftLayer. 
+
+> **Warning:** To fully enable virtual guest pooling on SoftLayer, except deploying a pool server, you also need to make director to connect to it and enable pooling feature. Please refer to guide on [bosh-softlayer-cpi-release](https://github.com/cloudfoundry-incubator/bosh-softlayer-cpi-release).
+
+
+
+
+> **Note:** 
+> In bosh CLI v2, bosh-init is deprecated and use `bosh create-env` instead.
```
Comment (maximilien) [inline README.md]: mention also that this doc will be updated.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+- To install bosh-init, please refer to [install-bosh-init](http://bosh.io/docs/install-bosh-init.html) and its usage can be found in [using-bosh-init](http://bosh.io/docs/using-bosh-init.html).
+> **Note:**
+>  Please make sure the machine installed bosh-init can access SoftLayer private network and you can enable SoftLayer VPN if it is outside of SoftLayer data center. This is because it need to communicate with the target VM over SoftLayer private network to accomplish a successful deployment.
```
Comment (maximilien) [inline README.md]: change "This is because it need" to "This is because it needs"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+You can find a deployment manifest example under docs named [vps-init-example.yml](https://github.com/cloudfoundry-community/bosh-softlayer-pool-server-release/tree/develop/docs) which can deploy a virtual guest pooling server and please replace release, stemcell, resource and credential information accordingly.
+> **Note:**
+>  For releases and stemcells, please either use url like the example manifest does or download them to your local machine and specify its location.
+>  
+>  - bosh-softlayer-pool-server-release
+>  - postgres-release
+>  - bosh-softlayer-cpi-release
+>  - SoftLayer light stemcell
+
```
Comment (maximilien) [inline README.md:43]: any way to add an example of the command that user would typically use. That should make their life a bit easier by cut and pasting?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+Here is an example for key properties of jobs.
+```
```
Comment (maximilien) [inline README.md:45]: I would also put a comment on any line where the user should change some value. Thx

File: README.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        db_host: 127.0.0.1
+        db_port: 5432
+        db_schema: bosh
+        db_driver: postgres
+
+```
+- Kick-off deployment. You will got an output during deploying as below.
+
+```
+bosh-init deploy <your-manifest.yml>
```
Comment (maximilien) [inline README.md]: Use `$ bosh-init deploy <your-manifest.yml>` to make clear that this is a command a the prompt ($)

**Review by maximilien — CHANGES_REQUESTED** [review]
Grace, this is much improved. Made a few line comments and few required comments. Please try to address all and resubmit and I will merge. Thx.

**Comment by zhanggbj** [comment]
Hi dr.max @maximilien ,

All comments addressed. Thank you.

**Review by maximilien — APPROVED** [review]
Looks good. Merging.

Let's make sure to keep this up to date as things change, especially when BMS is added.

**Pull request merged**